projects/storefrontlib/src/lib/cms/store/effects/navigation-entry-item.effect.spec.ts

```diff
@@ -34,21 +34,12 @@ describe('Navigation Entry Items Effects', () => {
   let service: OccCmsService;
   let effects: fromEffects.NavigationEntryItemEffects;
 
+  const components: Component[] = [
```

Comment:
this mocked data looks not good. Please use the old data.

Reply:
Ok. But,`Component` type doesn't have `url`, `linkName`, `target` fields. So the type of the components will be `any[]`